ci/env/kava-internal-testnet/genesis.json

```
@@ -1326,6 +1305,10 @@
           {
             "kava_erc20_address": "0x864121A3b64A05d872883a1D0DA04eEe107A64F3",
             "denom": "erc20/multichain/usdc"
+          },
+          {
+            "kava_erc20_address": "0x864121A3b64A05d872883a1D0DA04eEe107A64F3",
```

```suggestion
            "kava_erc20_address": "0x818ec0A7Fe18Ff94269904fCED6AE3DaE6d6dC0b",
```

https://explorer.kava.io/address/0x818ec0A7Fe18Ff94269904fCED6AE3DaE6d6dC0b/transactions